Userland/Libraries/LibGUI/JsonArrayModel.cpp

```diff
@@ -11,6 +11,8 @@
 
 namespace GUI {
 
+JsonArrayModel::~JsonArrayModel() = default;
```

Comment:
looking at this, the destructor was not the key function before, `data(ModelIndex const&, ModelRole)` was. So if the destructor is not emitted into `libgui-serenity.so` with gcc 15, that's a boog in the compiler and/or linker.

Reply:
That being said. the patch as-is is fine and benign and doesn't actually change any behavior -- explicitly defaulting a destructor at the declaration site has no effect if the thing is actually virtual.

Reply:
Does the symbol not exist at all, or is it just hidden?